outport/process/executionOrder/transactionsExecutionOrder.go

```diff
@@ -121,9 +127,9 @@ func (s *sorter) sortTransactions(transactions []data.TransactionHandlerWithGasU
 	}
 }
 
-func (s *sorter) extractNormalTransactionsAndInvalidFromMe(
+func (s *sorter) extractTransactionsGroupedFromMe(
 	pool *outport.Pool, blockBody *block.Body, header data.HeaderHandler, scheduledMbsFromPreviousBlock []*block.MiniBlock,
-) ([]data.TransactionHandlerWithGasUsedAndFee, []data.TransactionHandlerWithGasUsedAndFee, []string, error) {
+) (resultsTransactionsFromMe, error) {
```

Comment:
Why not `*resultsTransactionsFromMe` and `return nil, err` in L159, to avoid returning two not nil values, and `return &resultsTransactionsFromMe{` in L169

Reply:
refactored.